tools/build/extract-react-props.ts

```diff
@@ -0,0 +1,145 @@
+import {
```

Comment:
I wanna keep this for reference for future work when I mutate the typedoc exports into soemthing website prop table friendly.